### DISCUSSION TITLE
I can not do it

### DISCUSSION THREAD

**Discussion by Flodo1987** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#502] [Q&A]
Hi
I hope I'm right here.
I tried to integrate my (Roborock S50 with Valetudo) card into Home Assistant.
I can't get it to work, I've tried all the options I know of and changed browsers. Maybe someone can help me here?
![Screenshot (1)](https://user-images.githubusercontent.com/3922655/205157167-e28bd7de-2b3f-4f00-a9fd-f132255034fb.png)

**Comment by PiotrMachowski** [comment]
Open the link to documentation from the editor

**Reply by Flodo1987** [reply]
OK thanks
I'll try to understand what I have to do with it there, I had already looked at it, but my head couldn't do anything with it.

**Reply by PiotrMachowski** [reply]
Valetudo integration with the map card isn't perfect, I plan to improve it, but at this moment I don't have enough time.

**Comment by Flodo1987** [comment]
No problem with most of the times it works, it seems like I can't get it because I don't quite understand where I have to edit something to get it to work.

**Reply by PiotrMachowski** [reply]
The main part is that you need to install [I can't believe it's not Valetudo](https://github.com/Hypfer/Icantbelieveitsnotvaletudo)

**Reply by Flodo1987** [reply]
oh man, thank you for your tips but I can't even get it to work I think I'm missing the no-how behind it

I can't get the alternative (valetudo lovelace map card) that I found to work either
